docs/couchbase_columnar_api/results.rst

```diff
@@ -0,0 +1,28 @@
+==============
+Results
+==============
+
+.. module:: couchbase_columnar.result
+
+QueryResult
```

Comment:
Do we need `QueryResult` in the docs, if it's not a class we expect users to use directly?

Reply:
Fair point, removed in latest updated 👍